src/PlutoTeachingTools.jl

```diff
@@ -1,19 +1,56 @@
 module PlutoTeachingTools
 
-using PlutoUI
-using HypertextLiteral  # used by aside & present, might be useful elsewhere
-using Markdown
+using PlutoUI: Resource, LocalResource, combine
+using Markdown: @md_str, MD, Admonition, LaTeX
+using HypertextLiteral: @htl, @htl_str
+using Latexify: latexify
+using Downloads: download
```

Comment:
What about adding comments to each line, so we can easily remember which files actually make use of these packages?

Reply:
Sure, will do!

Reply:
Added in 9a9eae1.